src/main/scala/org/opennetworkinsight/netflow/FlowPostLDA.scala

```diff
@@ -19,112 +17,125 @@ object FlowPostLDA {
                   outputDelimiter: String,
                   threshold: Double, topK: Int,
                   ipToTopicMixes: Map[String, Array[Double]],
-                  ordToProbPerTopic : Map[String, Array[Double]],
+                  wordToProbPerTopic: Map[String, Array[Double]],
                   sc: SparkContext,
                   sqlContext: SQLContext,
                   logger: Logger) = {
 
-    var ibyt_cuts = new Array[Double](10)
-    var ipkt_cuts = new Array[Double](5)
-    var time_cuts = new Array[Double](10)
-
     logger.info("loading machine learning results")
 
-    val topics = sc.broadcast(ipToTopicMixes)
-
-
-    val words = sc.broadcast(ordToProbPerTopic)
-
     logger.info("loading data")
-    val rawdata: RDD[String] = {
-      val flowDataFrame = sqlContext.parquetFile(inputPath)
+    val totalDataDF: DataFrame = {
+      sqlContext.read.parquet(inputPath)
         .filter("trhour BETWEEN 0 AND 23 AND  " +
           "trminute BETWEEN 0 AND 59 AND  " +
           "trsec BETWEEN 0 AND 59")
-        .select("treceived",
-          "tryear",
-          "trmonth",
-          "trday",
-          "trhour",
-          "trminute",
-          "trsec",
-          "tdur",
-          "sip",
-          "dip",
-          "sport",
-          "dport",
-          "proto",
-          "flag",
-          "fwd",
-          "stos",
-          "ipkt",
-          "ibyt",
-          "opkt",
-          "obyt",
-          "input",
-          "output",
-          "sas",
-          "das",
-          "dtos",
-          "dir",
-          "rip")
-      flowDataFrame.map(_.mkString(","))
+        .select(Schema.TimeReceived,
+          Schema.Year,
```

Comment:
what do you think of doing an import FlowSchema._  and then using Month, Duration, SourcePort etc as constants throughout the file?  

since the only schema names being used are the Flow ones, there's no confusion and it cuts back on a lot of redundant notation